controller/namespace/namespacecontroller.go

```diff
@@ -28,21 +35,21 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
-// NamespaceController holds the connections for the controller
-type NamespaceController struct {
+// Controller holds the connections for the controller
+type Controller struct {
 	NamespaceClient        *rest.RESTClient
 	NamespaceClientset     *kubernetes.Clientset
 	Ctx                    context.Context
-	ThePodController       *PodController
-	TheJobController       *JobController
-	ThePgpolicyController  *PgpolicyController
-	ThePgreplicaController *PgreplicaController
-	ThePgclusterController *PgclusterController
-	ThePgtaskController    *PgtaskController
+	ThePodController       *pod.Controller
+	TheJobController       *job.Controller
+	ThePgpolicyController  *pgpolicy.Controller
+	ThePgreplicaController *pgreplica.Controller
+	ThePgclusterController *pgcluster.Controller
+	ThePgtaskController    *pgtask.Controller
```

Comment:
If you wanted to remove the `The` from the front of these, I wouldn't say no...

Reply:
`The` has been removed from these vars